schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl

```diff
@@ -280,9 +280,9 @@ protocol CommonParticipant {
     clinicalEthnicities supersedes Ancestries in GMS
     */
 
-    enum GenericConsent {`true`, `false`, not_discussed}
+    enum GenericConsent {`true`, `false`, undefined, not_applicable}
```

Comment:
if both of this have the same values, why do we need two objects? Have one object called `Consent` and name the fields accordingly

Reply:
Also, should we be explicit and be as identical to the consent service responses as possible and use `yes`, `no` instead of true/false?

Reply:
Done